src/diffusers/models/resnet.py

```diff
@@ -674,6 +674,34 @@ def forward(self, inputs):
         return output
 
 
+class MixingResidualBlock(nn.Module):
```

Comment:
Let's add a comment that this is used in Wuerstchen please

Reply:
Done